content/news/028/index.md

```diff
@@ -218,6 +218,37 @@ November's full weekly devlogs: "This Week In Veloren...":
 [veloren-reading-club-2]: https://www.youtube.com/watch?v=n8XayRvVBEs
 [veloren-rain-storm]: https://www.youtube.com/watch?v=MZwfaohynvc
 
+### [Hydrofoil Generation]
+
+![hgs_screen](hgs5.png)
+
+[Hydrofoil Generation] ([Facebook](https://www.facebook.com/HydrofoilGenerationSailing),
+[Discord](https://discord.gg/DtKgt2duAy)) is a realistic sailing/foiling
+inshore simulator in development for PC/Steam that will put you in the
+driving seat of modern competitive sailing.
+
+November was dedicated to the launch of the [Steam Store][hgs_steam] page and associated
+[trailer][hgs_trailer] showcasing Hydrofoil Generation's custom Rust engine in motion
+for the first time. Your chance to Wishlist a Rust made engine on Steam is now
+a reality.
```

Comment:
I'm not sure about this line, as it sort of makes it sound like there aren't other Rust games on steam (afaik there are a few)

Reply:
You are right, I took it out.